dags/events_to_amplitude.py

```diff
@@ -41,7 +41,22 @@
         "max_requests": FOCUS_ANDROID_INSTANCES * VCPUS_PER_INSTANCE,
         "key_file": key_file,
         "artifact": url,
-        "config_filename": config_file
+        "config_filename": "focus_android_events_schemas.json",
+    },
+    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/events_to_amplitude.sh",
+    dag=dag)
+
+savant_events_to_amplitude = EMRSparkOperator(
+    task_id="savant_events_to_amplitude",
+    job_name="Firefox Desktop Events to Amplitude",
+    execution_timeout=timedelta(hours=8),
+    instance_count=SAVANT_INSTANCES,
+    env={
+        "date": "{{ ds_nodash }}",
+        "max_requests": SAVANT_INSTANCES * VCPUS_PER_INSTANCE,
+        "key_file": key_file,
```

Comment:
You'll need to change the `key_file` locations. We should probably add another directory, e.g. `s3://telemetry-airflow/config/amplitude/focus_android/{}/apiKey` instead of `s3://telemetry-airflow/config/amplitude/{}/apiKey`.